js/apps/system/_admin/aardvark/APP/frontend/js/templates/modalTable.ejs

```diff
@@ -96,12 +102,13 @@ function createInput(row) {
                 <% }); %>
                 <td>
                             <span style="display: inline-flex">
-                                <button class="graphViewer-icon-button gv-icon-small add addAfter"></button>
                                 <% if (idx) {%>
                                 <button style="margin-left: 5px;"
                                         class="graphViewer-icon-button gv_internal_remove_line gv-icon-small delete addDelete">
                                 </button>
                                 <% } %>
+                                <% //<i class="moveRowButton moveRowUp disabled fa fa-arrow-circle-up"></i>
+                                // <i class="moveRowButton moveRowDown disabled fa fa-arrow-circle-down"></i> %>
```

Comment:
```suggestion
```

Reply:
Is there an alternative way to move rows up/down? Or was this requirement removed?

Reply:
Due to technical issues with the old framework, this feature should be added in our upcoming refactoring using the react js framework. This feature is planned for 3.10. The required time for implementing this feature in 3.9 (using backbone) only, and then drop it in 3.10 anyway, is not worth it. Additionally, a 3.9 backbone based implementation would have been super ugly, from a technical point of view. Also, the required time is out of proportion compared to the benefits it would deliver here.